File: Client-App/_model/fixture.ts
import { Team } from "./team"

export interface Fixture{
  id: number
  roundID: number
  homeID: number
  awayID: number
  homeResult: string
  awayResult: string
  resultReset: boolean
  timeStamped: Date
  home: Team
  away: Team
}

File: Web-Api/Model/Entities/Fixture.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace Pickfc.Model.Entities
{
    public class Fixture
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int ID { get; set; }
        public int RoundID { get; set; }
        public int  HomeID { get; set; }
        public int AwayID { get; set; }
        public string HomeResult { get; set; } = string.Empty; 
        public string AwayResult { get; set; } = string.Empty;
        public DateTime TimeStamped { get; set; }
    }
}
